packages/test-utils/src/jgd-wrapper.js

```diff
@@ -38,8 +38,28 @@ function configureJimp() {
   });
 }
 
-const Jimp =
-  typeof window !== "undefined" && window.Jimp ? window.Jimp : configureJimp();
+/**
+ * The ultimate hack
```

Comment:
Could you explain what you think is going on?

Reply:
Essentially this is just recursively unwrapping Jimp to get Jimp class. The reaason why I made it recursive is because I do not know what kind of updates you will make so this is a dirty future proof hack

Reply:
Could you add that in the comment?